blocks/form/checkbox.js

```diff
@@ -0,0 +1,30 @@
+function update(checkboxgroup) {
+  const allUnchecked = checkboxgroup.every((checkbox) => !checkbox.checked);
```

Comment:
you can access all the checkboxes using `document.getElementsByName(e.target.name)`

Reply:
same named checkbox already stored in closure, passing them